macros/clone_schema.sql

```diff
@@ -0,0 +1,266 @@
+{%- macro clone_schema(schema_one, schema_two, comment_tag='') -%}
```

Comment:
I've found that Postgres doesn't support tags mechanism as [Snowflake](https://docs.snowflake.com/en/user-guide/object-tagging.html) does so to make this macro more general I decided to use metadata field `comment` to filter objects we need to have.
I believe that only we would need to do for using this macro after this PR are:
1. Mark incremental tables and sequences in `dwh` by special comment 'incremental'.
2. Use command `dbt run-operation clone_schema --args '{schema_one: prod, schema_two: stage, comment_tag: incremental}' --target ANALYSIS`